src/entities/identity/lib/nameUtils.ts

```diff
@@ -0,0 +1,10 @@
+export function getFirstName(fullName: string): string {
```

Comment:
The function itself looks like workaround, BE should return splitted data

Reply:
That's exactly what it is. Yes, the back-end should return first and last names as independent fields